saleor/graphql/attribute/mutations.py

```diff
@@ -864,5 +867,9 @@ def perform_mutation(cls, _root, info, attribute_id, moves):
         with traced_atomic_transaction():
             perform_reordering(values_m2m, operations)
         attribute.refresh_from_db(fields=["values"])
+
+        for value in values_m2m.filter(id__in=operations.keys()):
```

Comment:
Do we need to query the DB here? Maybe filtering in Python would be enough. Not sure how much data we can have here.

Reply:
`values_m2m` is a `RelatedManager` object so I cant use python `filter()` on it

Reply:
I was thinking about sth similar to this:
```
for value in [v for v in values_m2m if v.id in operations.keys()]
```